required/latex-lab/testfiles-block/blocks-000.tlg

```diff
@@ -10,13 +10,13 @@ Package: latex-lab-kernel-changes ....-..-.. v... General kernel and class chang
 LaTeX template Info: Declaring template type 'blockenv' taking 1 argument(s)
 (template)           on line ....
 LaTeX template Info: Declaring template type 'block' taking 1 argument(s) on
-(template)           line 88.
+(template)           line 62.
 LaTeX template Info: Declaring template type 'para' taking 1 argument(s) on
-(template)           line 89.
+(template)           line 63.
 LaTeX template Info: Declaring template type 'list' taking 1 argument(s) on
-(template)           line 90.
+(template)           line 64.
 LaTeX template Info: Declaring template type 'item' taking 1 argument(s) on
-(template)           line 91.
+(template)           line 65.
```

Comment:
I guess we should look to normalise this for the future in `l3build`?

Reply:
maybe better to change the template typout to have a `\MessageBreak` so that it reads
```
LaTeX template Info: Declaring template type 'para' 
(template)          taking 1 argument(s) on line 63
```

Reply:
Sure, but the line number would still be there

Reply:
would it? isn't the string "on line xxx" recognized and replaced, because that would otherwise show all over the place.

Reply:
Ah true: I’ll adjust

Reply:
These auto-wrapped messages are results of `\iow_wrap:nnnN`, using the `\l_iow_line_count_int` ([defaults to 78](https://github.com/latex3/latex3/blob/1a138b7d6363c1a29c370450ac9c12bd2c3bbe88/l3kernel/l3file.dtx#L1938)). Maybe `l3build` should also set `\l_iow_line_count_int` to 9999, to be aligned with the `maxprintline = 9999` setting.

Reply:
Oh, `l3build` already sets `\l_iow_line_count_int` to 9999 since its 2024-02-08 release, see https://github.com/latex3/l3build/commit/fd19f1dcd0edfbb72606d431893964f521333874. The remaining work is to sync the `./support/regression-test.tex` local to this repo with that from `l3build`.

Reply:
Resolved in PR https://github.com/latex3/latex2e/pull/1490. Now in this repo l3msg lines use a 9999 `\l_iow_line_count_int`, the same as the default `l3build` settings.